### PR TITLE
Python 3.7.3 release

### PR DESCRIPTION
**Pull request by CaseyFaist**
A new Python binary is available on the Heroku stack! 

Plus some typo corrections in changelog and more test updates

### CODE REVIEW

File: test/run [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Default Python Versions
 # shellcheck source=bin/default_pythons
-source "$BIN_DIR/default_pythons"
+source "bin/default_pythons"
```
Comment (jkutner) [inline test/run]: was this on purpose?

Comment (CaseyFaist) [reply test/run]: It was, the file wasn't being found before the change. should I have changed the shellcheck instead?

Comment (jkutner) [reply test/run]: oh, so this just never worked before?

Comment (CaseyFaist) [reply test/run]: Yep, missed its effects in this test script due to false positives. I checked that it worked this time by running tests for all stacks with only one binary generated, it responds correctly to the failure case now

**Pull request merged**